spec/plaindatetime.html

```diff
@@ -1091,22 +1073,7 @@ <h1>CreateTemporalDateTimeFromStatic ( _constructor_, _year_, _month_, _day_, _h
       <h1>ToTemporalDateTimeFields ( _temporalDateTimeLike_, _fieldNames_ )</h1>
       <emu-note>The value of ? ToIntegerOrInfinity(*undefined*) is 0.</emu-note>
```

Comment:
This note is no longer relevant here, but I'd add it to PrepareTemporalFields instead.

Reply:
Should ToPositiveIntegerOrInfinity convert *undefined* to 1, or throw a RangeError? The latter is the spec behavior in this PR.

Reply:
I think the latter, but the note still applies to the optional time properties.